modalities/dom/components/elements/model-img.js

```diff
@@ -11,33 +11,45 @@ subject to an additional IP rights grant found at http://polymer.github.io/PATEN
 import Xen from '../xen/xen.js';
 
 const log = Xen.logFactory('ModelImg', 'blue');
+/*
+ Has two modes:
+  add an <img> child and set `src`
+  set `url` (will use background style)
+
+ Probably should be two elements instead.
+*/
 
 class ModelImg extends Xen.Base {
   static get observedAttributes() {
     return ['src', 'url', 'fadems'];
   }
   _update({src, url, fadems}, state) {
     const fade = fadems || 150;
-    const img = this.img;
-    if (img && !state.listener) {
-      state.listener = img.addEventListener('load', e => this.onLoad(img, e));
-    }
-    if (img && src) {
+    if (src && state.src !== src) {
+      //log(src);
```

Comment:
Delete log() here and below?

Reply:
model-img still needs work, so I left some stuff in there.